File: Blackjack21-JAVA/Blackjack21Library/src/Blackjack21/Game/Exceptions/PlayerNotFoundException.java
package Blackjack21.Game.Exceptions;

/**
 * Throws a player not found exception based on an invalid index
 */
public class PlayerNotFoundException extends Exception
{

    public PlayerNotFoundException(int index)
    {
        super("Index " + index + " is not a valid player entry");
    }

    public PlayerNotFoundException(String playerName)
    {
        super("Player " + playerName + " not found");
    }

}

File: BlackJack-CSHARP/Blackjack21.Game/Game/Exceptions/PlayerNotFoundException.cs
using System;

namespace Blackjack21.Game.Exceptions
{
    /// <summary>
    /// Throws a player not found exception based on an invalid index
    /// </summary>
    public class PlayerNotFoundException : Exception
    {
        public PlayerNotFoundException(int index) : base(string.Format("Index {0} is not a valid player entry", index))
        {

        }

        public PlayerNotFoundException(string playerName) : base(string.Format("Player {0} not found", playerName))
        {

        }


    }
}
